kong/init.lua

```diff
@@ -199,16 +199,16 @@ end
 
 
 local function update_plugins()
-  local version, err = kong.cache:get("plugins_map:version", TTL_ZERO, utils.uuid)
+  local version, err = kong.cache:get("plugins:version", TTL_ZERO, utils.uuid)
```

Comment:
Was there a practical necessity for this? I ask because cache keys for entities are often built as `<entity>:<data>`, so this ends up looking like a regular entity key (storing a `plugins` entity value), rather than a special key. I'd rather keep the `plugins:` namespace reserved for the `plugins` DAO.

Reply:
plugins contains:
1. plugins.map
2. plugins.combos
3. plugins.loaded
4. plugins.phases
(and originally plugins.configured, which we dropped)

So plugins_map doesn't seem like a descriptive or best name for it anymore. Perhaps just call it plugins.version or plugins/version or plugins#version. But then it differs from router:version.